pkg/kubeapiserver/options/authentication.go

```diff
@@ -283,6 +295,11 @@ func (s *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 			"Identifier of the service account token issuer. The issuer will assert this identifier "+
 			"in \"iss\" claim of issued tokens. This value is a string or URI.")
 
+		fs.StringVar(&s.ServiceAccounts.JWKSURI, "service-account-jwks-uri", s.ServiceAccounts.JWKSURI, ""+
```

Comment:
Why is this configurable? Can we pick a path?

Reply:
We'll want to override it if serving cached responses from a bucket, for example.